pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -684,6 +684,34 @@ def aggfunc(x):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("func", ["min", "max"])
```

Comment:
can you move these to test_timegrouper.py (same dir)

Reply:
Since we're grouping by an int column and only aggregating the period column (not using a time grouper), does the current location make more sense?

Reply:
This location seems OK since we're grouping by integers, not time.